src/components/sidebar/right-sidebar/Filters.tsx

```diff
@@ -1,21 +1,41 @@
 import React, { useContext } from "react";
 import { AppContext } from "../../../context";
 import { Dropdown } from 'semantic-ui-react'
-import {FilterType} from '../../../types';
+import { FilterType } from '../../../types';
 import InformationIcon from "../../shared/InformationIcon";
+import Translate from "../../../utils/translate/translateService";
+import { toPascalCase } from "../../../utils/translate/caseChanger";
+import { getCountryName } from "../../../utils/mapUtils";
 
 export default function Filters() {
   const [state, dispatch] = useContext(AppContext);
 
-  const formatOptions = (options: any) => {
+  const formatOptions = (options: any, filter_type: FilterType) => {
     const formatted_options: Record<string, string>[] = [];
-    options.forEach((o: string) => {
-      formatted_options.push({
-        key: o,
-        text: o,
-        value: o
-      })
-    });
+    const optionString = toPascalCase(filter_type.toString());
+    const jsonObjectString = `${optionString}Options`;
+    switch (filter_type) {
+      case 'country':
+        options.forEach((o: string) => {
+          formatted_options.push({
+            key: o,
+            text: getCountryName(o, state.language, jsonObjectString),
+            value: o
+          })
+        });
+        break;
+      default:
+        options.forEach((o: string) => {
+          const translatedString = Translate(jsonObjectString, [toPascalCase(o)]);

```

Comment:
why pascal case?

Reply:
because you need to turn them into that to access the json objects